projects/ffmpeg/run_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd $SRC/ffmpeg
+
+# TODO: Since the h264dsp test is failing, we are using a patch to skip it
+# entirely. Although adding --ignore-tests=checkasm-h264dsp to ./configure is an
+#option, it still compiles the test and wastes a lot of time.
```

Comment:
From
```
real    22m49.004s
user    11m7.424s
sys     14m57.508s
```
to
```
real    0m43.515s
user    2m25.838s
sys     1m18.607s
```

Reply:
oh wait, I misread this. this is seconds, not minutes. ignore!

Reply:
It is minutes, from over 22 minutes to less than 1.